pycontw/pycon.tw
# Adjust talk list and detail page to 2020 style

Also expose the talk list from events menu and hide the cfp menu.

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies

- [x] **Other (page style)**

## Description
**Describe what the change is**
Adapt the 2020 styles for talk list and detail pages. Also expose the page to the menu.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to 'Events' -> 'Talks'
2. Click on any talk link to see the detail page

## Expected behavior
You should see the adapted styles in the Talks (1) and detail page (2).

## Related Issue
If applicable, refernce to the issue related to this pull request.

## More Information
**Screenshots**
<img width="835" alt="圖片" src="https://user-images.githubusercontent.com/484883/85971355-75a88980-b9ff-11ea-8986-0dfe380aa80a.png">


**Additional context**
Thank god the icons are in SVGs so I can change the color without opening any image processing applications :D
